### DISCUSSION TITLE
Theme in Windows

### DISCUSSION THREAD

**Discussion by YagoMaia** [eza-community/eza-themes#7] [Q&A]
I would be very grateful if you could put the themes on windowns. I tried a few ways, but it didn't work.

**Comment by BanHammerYKT** [comment]
Just add `EZA_CONFIG_DIR` environment var and put `theme.yml` in

I use powershell, and in my case I add into `$PROFILE.CurrentUserAllHosts` file
```powershell
$env:EZA_CONFIG_DIR = "$env:USERPROFILE\.config\eza"
```

Also I replaced `ls` built-in alias with this code
```powershell
Remove-Item Alias:ls -Force; New-Alias ls eza
```

**Comment by rizard1** [comment]
i'm a beginner i don't know lot about powershell can you show me step by step to add that config file at least with pictures.

**Comment by BanHammerYKT** [comment]
1. firstly open powershell, the commands below paste to them
2. create directory `%userprofile%\.config\eza` with this command:
`New-Item -ItemType Directory -Path "$env:USERPROFILE\.config\eza" -Force`
3. open explorer in created directory with this command:
`Start-Process explorer.exe "$env:USERPROFILE\.config\eza"`
5. download preferred theme yaml-file into created directory & rename to `theme.yml`
6. open powershell config file with this command:
`notepad $PROFILE.CurrentUserAllHosts`
7. paste code below into opened config-file (warn, any code in this file execute on every powershell start)
```
# set eza config directory
$env:EZA_CONFIG_DIR = "$env:USERPROFILE\.config\eza"

# optional
# replace `ls` built-in alias with eza
Remove-Item Alias:ls -Force; New-Alias ls eza
```
8. restart powershell
9. profit!!!
<br><br><br><br>
<details>
  <summary>some screenshots</summary>


![powershell](https://github.com/user-attachments/assets/75381e79-9a45-4ed6-ba31-aa3b6d7de62e)
![notepad](https://github.com/user-attachments/assets/ba474503-7ec2-46e4-83d7-dbd27d8af013)
![explorer](https://github.com/user-attachments/assets/0d56ce2d-720e-4d2c-a264-76ccbb379aa9)

  
</details>

**Reply by Salvodif** [reply]
Hi,

I followed your guide but:

```powershell
 salva@Glamdring    ll
ll : The term 'll' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included,
verify that the path is correct and try again.
At line:1 char:1
+ ll
+ ~~
    + CategoryInfo          : ObjectNotFound: (ll:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Anyidea?
BTW my eza directory is 

`C:\Users\salva\AppData\Local\Microsoft\WinGet\Packages\eza-community.eza_Microsoft.Winget.Source_8wekyb3d8bbwe`

> 1. firstly open powershell, the commands below paste to them
> 2. create directory `%userprofile%\.config\eza` with this command:
>    `New-Item -ItemType Directory -Path "$env:USERPROFILE\.config\eza" -Force`
> 3. open explorer in created directory with this command:
>    `Start-Process explorer.exe "$env:USERPROFILE\.config\eza"`
> 4. download preferred theme yaml-file into created directory & rename to `theme.yml`
> 5. open powershell config file with this command:
>    `notepad $PROFILE.CurrentUserAllHosts`
> 6. paste code below into opened config-file (warn, any code in this file execute on every powershell start)
> 
> ```
> # set eza config directory
> $env:EZA_CONFIG_DIR = "$env:USERPROFILE\.config\eza"
> 
> # optional
> # replace `ls` built-in alias with eza
> Remove-Item Alias:ls -Force; New-Alias ls eza
> ```
> 
> 8. restart powershell
> 9. profit!!!
> 
> some screenshots

**Comment by gierens** [comment]
Since this is more a question than an issue, I converted this to a discussion. ... But maybe we could also add some documentation for Windows in the Installation section of the README. @BanHammerYKT since you wrote the guide here, do you wanna create a PR for that?

**Comment by gierens** [comment]
I added a note to the installation section in the readme for powershell, hope this helps others trying to get this to work on Windows.

**Comment by ningw42** [comment]
Just a note for those who may have trouble as I am as of 0.20.18, putting your `theme.yml` to `~\AppData\Roaming\eza` should work too.

Windows doesn't follow the XDG_CONFIG_HOME convention, which means `~\.config\eza` isn't the default location for `eza`'s config files, but `~\AppData\Roaming\eza`. That's why you need `$env:EZA_CONFIG_DIR = "$env:USERPROFILE\.config\eza"` to override the default config file resolution if you put your `theme.yml` there.
